src/components/ProjectName/ProjectName.jsx

```diff
@@ -47,57 +48,68 @@ const BiggerIconGlobe = styled(IconGlobe)`
 `
 
 const ProjectName = () => {
-  const [projectName, setProjectName] = useState('')
-  const [isTestProject, setIsTestProject] = useState(false)
   const isMounted = useIsMounted()
   const { projectId } = useParams()
   const { databaseSwitchboardInstance } = useDatabaseSwitchboardInstance()
+  const { isAppOnline } = useOnlineStatus()
   const { currentUser } = useCurrentUser()
   const { mermaidExploreLink, isExploreLaunchEnabledForUser } = useExploreLaunchFeature({
     currentUser,
   })
+  const [project, setProject] = useState({})
 
   const _getProjectName = useEffect(() => {
     if (databaseSwitchboardInstance) {
       databaseSwitchboardInstance.getProject(projectId).then((projectResponse) => {
         if (isMounted.current) {
-          setProjectName(projectResponse?.name)
-          setIsTestProject(projectResponse?.status < 90)
+          setProject(projectResponse)
         }
       })
     }
   }, [databaseSwitchboardInstance, isMounted, projectId])
 
   const handleExploreButtonClick = () => {
-    if (!projectName) {
-      toast.error(...getToastArguments(language.error.noProjectMermaidExplore))
-      return
+    const { name, bbox } = project
+    const queryParams = new URLSearchParams({ project: name })
+
+    if (bbox) {
+      queryParams.append('bbox', `${bbox.xmin},${bbox.ymin},${bbox.xmax},${bbox.ymax}`)
     }
 
-    window.open(`${mermaidExploreLink}/?project=${projectName}`, '_blank')
+    window.open(`${mermaidExploreLink}/?${queryParams.toString()}`, '_blank')
   }
 
-  const mermaidExploreButton = isExploreLaunchEnabledForUser ? (
-    <MuiTooltip title={language.pages.gotoExplore('this project')} placement="top" arrow>
-      <IconButton
-        type="button"
-        aria-label="View MERMAID Explore"
-        onClick={handleExploreButtonClick}
-      >
-        <BiggerIconGlobe />
-      </IconButton>
-    </MuiTooltip>
-  ) : (
-    <ProjectNameLink href={`${mermaidExploreLink}/?project=${projectName}`} target="_blank">
-      <IconGlobe />
-      <span>{language.pages.goToDashboard}</span>
-    </ProjectNameLink>
-  )
+  const renderExploreButton = () => {
+    const isTestProject = project?.status < 90
+
+    if (!isAppOnline || isTestProject) {
+      return null
+    }
+
+    const exploreButtonContent = isExploreLaunchEnabledForUser ? (
+      <MuiTooltip title={language.pages.gotoExplore('this project')} placement="top" arrow>
+        <IconButton
+          type="button"
+          aria-label="View MERMAID Explore"
```

Comment:
Request: Can this be a language token?

Reply:
I don't think we do it for `aria-label`

Reply:
We can start adding them in, they'll also need to be translated or updated. Any text that a user might see or work with should go into the translation file

Reply:
it's specifically used for accessibility, so I think we don't do it in the first place.
May be @alanjleonard has more say in this matter, do you want us to put all aria-label text inside language file?

I’m happy to take care of it, but I suggest handling the accessibility text updates in a separate ticket where I can move everything into the language files. That way, this ticket stays focused and doesn’t become too overwhelming.

Reply:
@tienhoah One of the long term plans is having all of the hard coded text to be tokenized, including aria-labels. It would be helpful if we are adding them as we go, with less overall effort and less error-prone than having one ticket to bulk update everything. Please add this string to the language file
```suggestion
          aria-label={language.userMessage.viewExplorer}
```

```language.tsx
const userMessage = {
 viewExplorer: 'View MERMAID Explore'
} 
```

Reply:
@hurrendor I added the separate object used for accessibility text, we could build that up more in the future.

Reply:
Wouldn't it be the same as the label which would be in the language file?

Reply:
@alanjleonard  I don't see this one existed in language file.

Reply:
@alanjleonard Are you asking under the assumption of having the same text for the tooltip as what would be on the ARIA help text?